README.md & LICENSE.md (#12)
fullstacktf/order-food-backend: README.md

<p align="center"><small>From September 2021 to December 2021</small></p>

<p align="center"><a href="#">View Demo</a> · <a href="https://github.com/fullstacktf/order-food-backend/issues">Report a Bug</a> · <a href="https://github.com/fullstacktf/order-food-frontend">Front-End Repository »</a></p>

## Index

<details>
<summary>Table of Contents</summary>
<ol>
    <li><a href="#about-comidit.app">About Comidit.App</a>
        <ul>
        <li><a href="#built-with">Built With</a></li>
        </ul>
    </li>
    <li>
        <a href="#getting-started">Getting Started</a>
        <ul>
            <li><a href="#prerequisites">Prerequisites</a></li>
            <li><a href="#installation">Installation</a></li>
        </ul>
    </li>
    <li><a href="#usage">Usage</a></li>
    <li><a href="#roadmap">Roadmap</a></li>
    <li><a href="#license">License</a></li>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#acknowledgements">Acknowledgments</a></li>
</ol>
</details>

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

## About Comidit.App

> <p align="center"><strong>Comidit.App makes your orders easy and fast, accompaning you in your everyday life.</strong></p>

Reviewer: ```suggestion
<p align="center"><strong>Comidit.App makes your orders easy and fast, accompanying you in your everyday life.</strong></p>

```

> <p align="center">We know there are a lot of websites where you could order your food but...Are you sure that they are THIS AMAZING? 💛 </p>

Reviewer: ```suggestion
<p align="center">We know there's a lot of websites where you could order your food but... Are you sure they are THIS AMAZING? 💛 </p>

```

<br/>

> Our goal is provide a free, easy, and responsive web application for restaurants who wants make easier the process of managing their orders in addition stay closer to their customers giving the best delivery service. Comidit.App is designed for people, providing an intuitive, effortless offering you the most valuable restaurants of your area.

Reviewer: ```suggestion
Our goal is to provide a free, easy, and responsive web application for restaurants who want to make the process of managing their orders easier, in addition to staying closer to their customers while offering the best delivery service. Comidit.App is designed for people, providing an intuitive, effortless offering of the most valuable restaurants of your area.

```

> Comidit.App is our final project of the Fullstack's Development course organized by the Industrial Business School in Spain. This project was realized since September to December 2021 with non commercial purposes.

Reviewer: ```suggestion
Comidit.App is our final project for the Fullstack's Development course organized by the Industrial Business School in Spain. This project was developed from September to December 2021 with non commercial purposes.

```

![img stripe](https://img.shields.io/twitter/url?color=blueviolet&label=stripe&logo=stripe&logoColor=white&style=for-the-badge&url=https%3A%2F%2Fgithub.com%2Ffullstacktf%2Forder-food-backend)

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

## Getting Started

### Prerequisites

To get a local copy up and running follow these simple steps:

``` npm
npm install npm@latest -g
```

### Installation

1. Clone the repository

``` npm
https://github.com/fullstacktf/order-food-backend.git
```

2. Install NPM packages

``` npm
npm install
```

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

## Usage

> We still working on Comidit.App until we finish you can [check our process](https://github.com/fullstacktf/order-food-backend/projects/1) (kanban board).

Reviewer: ```suggestion
We're still working on Comidit.App.  Meanwhile, you can [check our progress](https://github.com/fullstacktf/order-food-backend/projects/1) (kanban board).

```

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

## Roadmap

> According to our MVP we will develop this features. The not checked ones will be extra features that we would like to do.

Reviewer: ```suggestion
According to our MVP we will develop this features. The unchecked ones are extra features we would like to work on after.

```

## License

<p align="center"><img src="https://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png"/></p>

Distributed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">**Creative Commons BY-NC-SA**</a>. This license allows reusers to distribute, remix, adapt, and build upon the material in any medium or  format for noncommercial purposes only, and only so long as attribution  is given to the creator. If you remix, adapt, or build upon the  material, you must license the modified material under identical terms. CC BY-NC-SA includes the following elements:

<p align="center"><img alt="by img" src="https://mirrors.creativecommons.org/presskit/icons/by.png"/> <img alt="nc img" src="https://mirrors.creativecommons.org/presskit/icons/nc.png"/> <img alt="sa img" src="https://mirrors.creativecommons.org/presskit/icons/sa.png"/></p>

- **BY** - Credit must be given to the creator
- **NC** - Only noncommercial uses of the work are permitted

- **SA** - Adaptations must be shared under the same terms

Added to that we use [**MIT License**](LICENSE.md).

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

## Contact

About our team:

> - **Manuel** González - [GitHub · Profile](https://github.com/ManuYuzu)

Reviewer: No sé si maybe él no quiere ponerlo idk

```suggestion
- **Manuel** González Afonso - [GitHub · Profile](https://github.com/ManuYuzu)

```

> - **José** Nicolás Cabrera - [GitHub · Profile](https://github.com/jncabdom)

Reviewer: ```suggestion
- **Nicolás** Cabrera  Domínguez - [GitHub · Profile](https://github.com/jncabdom)

```

- **Paula** Pérez Tafalla -  [GitHub · Profile](https://github.com/paolaski) - [Paolaski.es · Website](https://www.paolaski.es/)

> - **Nestor** Torres Diaz - [GitHub · Profile](https://github.com/dtote)

Reviewer: ```suggestion
- **Néstor** Torres Diaz - [GitHub · Profile](https://github.com/dtote)

```

<small><p align="right">(<a href="#top">back to top</a>)</p></small>

> ## Acknowledgements

Reviewer: A qué te refieres con Acknowledgements? Si tal ponme como lo pensaste en español a ver si se me ocurre alternativa.
No sé por qué me suena medio raro

Reviewer 2: Se supone que es anotaciones extra, sería algo así como menciones. La [referencia](https://github.com/othneildrew/Best-README-Template)

Reviewer: ahhhmigo ya entendí el uso 

perfect tonses, todo completo
